community/kernel/src/test/java/org/neo4j/kernel/api/index/ArrayEncoderTest.java

```diff
@@ -41,11 +41,11 @@
     public final ThreadingRule threads = new ThreadingRule();
 
     @Test
-    public void WARNING_TheEncodingIsPlatformDependant_WARNING() throws Exception
+    public void encodingShouldNotContainNewLines() throws Exception
     {
         String encoded = ArrayEncoder.encode( new String[]{
                 "This string is long enough for BASE64 to emit a line break, making the encoding platform dependant."} );
-        assertTrue( "The encoding is platform dependant", encoded.contains( System.lineSeparator() ) );
+        assertFalse( "The encoding is platform dependant", encoded.contains( System.lineSeparator() ) );
```

Comment:
@martinfurmanski My gut feeling is "yes", although I think instead we should actually check all chars in the encoded string and verify that they are all members of the expected set of characters.